tokio/src/runtime/scheduler/multi_thread/worker.rs

```diff
@@ -787,6 +787,10 @@ impl Core {
                 cap,
             );
 
+            // Take at least one task since the first task is returned directly
+            // and nto pushed onto the local queue.
+            let n = usize::max(1, n);
```

Comment:
Seems like you should add one to `cap` in the `usize::min` above instead, no?

Reply:
I could... I thought it would be clearer to split it out w/ a comment.